src/main/java/org/candlepin/subscriptions/capacity/files/ProductDenylist.java

```diff
@@ -22,23 +22,33 @@
 
 import java.io.IOException;
 import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
 import java.util.Set;
+import java.util.stream.Stream;
 import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
 import org.candlepin.subscriptions.ApplicationProperties;
 import org.candlepin.subscriptions.files.PerLineFileSource;
 import org.candlepin.subscriptions.util.ApplicationClock;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.context.ResourceLoaderAware;
 import org.springframework.core.io.ResourceLoader;
 import org.springframework.stereotype.Component;
 import org.springframework.util.StringUtils;
 
-/** List of products to be considered for capacity calculations. */
+/** List of products to be skipped for capacity calculations. */
+@Slf4j
 @Component
 public class ProductDenylist implements ResourceLoaderAware {
+  private static final List<String> suffixes =
+      Stream.of("F2", "F3", "F3RN", "F4", "F5", "HR", "MO", "RN", "S")
```

Comment:
This seems like something that should be configurable. How likely is this to change?

Reply:
Highly unlikely to change, and we'd want to do as part of a release anyways, so don't care to make configurable.